protocols/gossipsub/src/handler.rs

```diff
@@ -431,15 +431,6 @@ impl ConnectionHandler for Handler {
                     return KeepAlive::Yes;
```

Comment:
If you are feeling fancy, we could now refactor this to use a match-arm guard:

```rust
match self {
	Handler::Enabled(h) if h.in_mesh => KeepAlive::Yes,
	_ => KeepAlive::No
}
```

Reply:
fixed 3e75728